.ci/check-for-chart-changes.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# check for chart changes to speedup ci
+#
+
+set -o errexit
+set -o pipefail
+
+CHART_REPO="https://github.com/kiwigrid/helm-charts.git"
+
+echo "Check for chart changes to speedup ci..."
+
+git remote add chart-changes "${CHART_REPO}"
+git fetch chart-changes master
+
+if [ -z "$(git diff --find-renames --name-only "$(git rev-parse --abbrev-ref HEAD)" remotes/chart-changes/master -- charts)" ]; then
```

Comment:
Couldn't the same (what the complete script does) be achieved with a GitHub Action containing the `paths` configuration?
https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#onpushpull_requestpaths

Reply:
yes, but in this cae you would'nt get an error message but your workflow would just not start. imho this would be confusing for the user.

Reply:
Ok, understood